docs/stdlib/array.rst

```diff
@@ -161,14 +159,13 @@ Reference
 
 .. eql:operator:: arrayslice: array<anytype> [ int64 : int64 ] -> anytype
 
-    Array slicing.
-
-    An omitted lower bound defaults to zero, and an omitted upper
-    bound defaults to the size of the array.
+    Slices an array of :eql:type:`anytype` containing :eql:type:`int64`.
```

Comment:
Let's try to explain slicing briefly. Suggestion:
> Produces an array containing a subset of the original array's elements.

This may not be 100% accurate since you could slice an array with all of the original's elements, but I would like to find a way to tell what the operator does rather than repeating the name of it. Open to suggestions.

Reply:
I'll give this a go, ideally I want to make a reference to that type and the character limit may pose an issue with that